daprdocs/content/en/concepts/dapr-services/operator.md

```diff
@@ -9,4 +9,34 @@ When running Dapr in [Kubernetes mode]({{< ref kubernetes >}}), a pod running th
 
 ## Running the operator service
 
-The operator service is deployed as part of `dapr init -k`, or via the Dapr Helm charts. For more information on running Dapr on Kubernetes, visit the [Kubernetes hosting page]({{< ref kubernetes >}}).
\ No newline at end of file
+The operator service is deployed as part of `dapr init -k`, or via the Dapr Helm charts. For more information on running Dapr on Kubernetes, visit the [Kubernetes hosting page]({{< ref kubernetes >}}).
+
+## Additional configuration options
+
+The operator service includes additional configuration options.
+
+### Injector watchdog
+
+The operator service includes an _injector watchdog_ feature which periodically polls all pods running in your Kubernetes cluster and confirms that the Dapr sidecar is injected in those which have the `dapr.io/enabled=true` annotation. It is primarily meant to address situations where the [Injector service]({{< ref sidecar-injector >}}) did not successfully inject the sidecar (the `daprd` container) into pods.
+
+
+The injector watchdog can be useful in a few situations, including:
+
+- Recovering from a Kubernetes cluster completely stopped. When a cluster is completely stopped and then restarted (including in the case of a total cluster failure), pods are restarted in a random order. If your application is restarted before the Dapr control plane (specifically the Injector service) is ready, the Dapr sidecar may not be injected into your application's pods, causing your application to behave unexpectedly.
+
+- Addressing potential random failures with the sidecar injector, such as transient failures within the sidecar injector service.
+
+If the watchdog detects a pod that does not have a sidecar when it should have had one, it deletes it. Kubernetes will then re-create the pod, invoking the Dapr sidecar injector again.
+
+The injector watchdog feature is **disabled by default**.
+
+You can enable it by passing the `--watch-interval` flag to the `operator` command, which can assume one of the following values:

```

Comment:
So the only way I can do this realistically is via Helm?

Reply:
Assuming you're using Helm, yes.

Otherwise, if you're running Dapr on Kubernetes with your own chart (not sure if anyone does that, but...), then you need to pass the flag to the command